dds_web/static/swagger.yaml

```diff
@@ -466,7 +466,339 @@ paths:
+      security:
+        - encryptedToken: []
+  /user/totp/activate:
+    post:
+      tags:
+        - user
+      summary: Request to switch from HOTP to TOTP for second factor authentication.
+      operationId: requestTOTPActivation
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: string
+                example: "{message: Please check your email and follow the attached link to activate two-factor with autenticator app}" # yamllint disable-line
+  /users:
+    get:
+      tags:
+        - user
+      summary: List unit users within the unit the current user is connected to, or the one defined by a superadmin.
+        CHECK METHOD FOR SUPERADMINS
+      description: TODO - for superadmin, it can be passed a unit, currently in body, needs to be in query.
+      deprecated: true
+      operationId: getUsers
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/UserList"
+  /user/invites:
+    get:
+      tags:
+        - user
+      summary: Provides a list of invited users.
+        Superadmins see all invites, unitusers the invites to their projects and units.
+      operationId: getInvites
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  invites:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/InvitesList"
+                  keys:
+                    type: array
+                    items:
+                      type: string
+                    example: [Email, Role, Projects, Created]
+  /usage:
+    get:
+      tags:
+        - user
+        - project
```

Comment:
How is this endpoint called? I couldn't find anything in dds_cli calling it, neither in commands.py

Reply:
Ok, I wasn't clear enough here: I meant what is making requests to this endpoint, which cli command or possibly backend command?

Reply:
Could happen that it is an endpoint created back in the day but forgotten. Could be deprecated in the future if we don't think it is good or usefull to have it